### PR TITLE
Add support for Fabric

### PR DESCRIPTION
**Pull request by Awakened-Redstone**
- [x] Tested placeholders
- [x] Tested in production
- [x] Tested the all in one JAR

This pull request adds support for Patbox's PlaceholderAPI on Fabric servers
Quit support is **unknown**

The mod doesn't include the placeholder API, it depends on the server having a mod that does, if used without another mod that uses it the server won't load due to missing dependency

The mod is server only, it is unsupported on clients

### CODE REVIEW

**Comment by WiIIiam278** [comment]
Java 17 is fine--but will the fat jar compiled here run on a 1.16.5 J16 server? Might be work testing. Excellent work in all though, will review when I get a chance :)

**Comment by Awakened-Redstone** [comment]
> Java 17 is fine--but will the fat jar compiled with it run on a 1.16.5 J16 spigot server? Might be work testing. Excellent work in all though, will review when I get a chance :)

Just did a test using J11, since the project compatibility is set to 16 it compiled using J16
I got that it was using `class file version 60.0`, and after a search it seems to correspond to J16

**Comment by WiIIiam278** [comment]
> > Java 17 is fine--but will the fat jar compiled with it run on a 1.16.5 J16 spigot server? Might be work testing. Excellent work in all though, will review when I get a chance :)
> 
> Just did a test using J11, since the project compile version is set to 16 it compiled using the J16 class version I got that it was using `class file version 60.0`, and after a search it seems to correspond to J16

Got it -- great.

File: fabric/build.gradle [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    maven { url 'https://maven.nucleoid.xyz' } //PlaceholderAPI
+}
+
+dependencies {
+    minecraft "com.mojang:minecraft:1.19.3"
+    mappings "net.fabricmc:yarn:1.19.3+build.5:v2"
+    modCompileOnly "net.fabricmc:fabric-loader:0.14.17"
+    modCompileOnly "net.fabricmc.fabric-api:fabric-api:0.76.0+1.19.3"
+
+    modCompileOnly "eu.pb4:placeholder-api:2.0.0-pre.3+1.19.3"
```
Comment (WiIIiam278) [inline fabric/build.gradle:15]:
```suggestion
    modCompileOnly "eu.pb4:placeholder-api:2.0.0-pre.3+1.19.3"
```
styyle/spacing

**Review by WiIIiam278 — APPROVED** [review]
Only a very minor thing before merge, but this is pretty much excellent! really solid work indeed :)
I guess my only other question is: wouldn't it make more sense to target 1.19.4 than .3 right now?

**Comment by WiIIiam278** [comment]
Lgtm now

**Pull request merged**